styles.css

```diff
@@ -0,0 +1,93 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.main {
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  background-image: url("background.png");
+  background-repeat: no-repeat;
+}
+
+.logo {
+  position: absolute;
+  top: 40px;
+  font-size: 18px;
+  font-weight: 700;
+  padding: 24px;
+  color: #6070ff;
+  text-decoration: none;
+}
+
+.navbar {
+  position: absolute;
+  top: 30px;
+  right: 30px;
+  padding: 36px;
+  color: #acb7c3;
+}
```

Comment:
- [ ] Please add the correct margin-top value to your header by decreasing it to match the given design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/170527163-7c59d671-3006-440e-9758-6b0642e628d7.png)  | ![image](https://user-images.githubusercontent.com/68796232/170527276-e02572ea-2aba-496d-be38-98b51726c09c.png) |